### PR TITLE
Added Cart page

### PR DESCRIPTION
**Pull request by talrejah02**
Please review and give suggestions here is the live [link](https://deploy-preview-4--idyllic-klepon-23d3cf.netlify.app/)

-Added cart page
-horizontal card

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ function App() {
         <Route path="/" element={<Home/>}/>
         <Route path="/Login" element={<Login/>}/>
         <Route path="/Shop" element={<ProductPage/>}/>
+        <Route path="/Cart" element={<Cart/>}/>
```
Comment (msdeshmukh009) [inline src/App.jsx:17]: You can make separate component for all the routes.

File: src/components/Cart-Item-card/CartitemCard.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                <svg xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink"
+                    aria-hidden="true" role="img" className="iconify iconify--ph" width="22" height="22"
+                    preserveAspectRatio="xMidYMid meet" viewBox="0 0 256 256">
+                    <path className="outline"
+                        d="M128 216S28 160 28 92a52 52 0 0 1 100-20a52 52 0 0 1 100 20c0 68-100 124-100 124z">
+                    </path>
+                    <path className="inside"
+                        d="M128 224a7.8 7.8 0 0 1-3.9-1C119.8 220.6 20 163.9 20 92a60 60 0 0 1 108-36a60 60 0 0 1 108 36c0 30.6-17.7 62-52.6 93.4a314.3 314.3 0 0 1-51.5 37.6a7.8 7.8 0 0 1-3.9 1zm-3.9-15zM80 48a44 44 0 0 0-44 44c0 55.2 74 103.7 92 114.7c18-11 92-59.5 92-114.7a44 44 0 0 0-84.6-17a8 8 0 0 1-14.8 0A43.8 43.8 0 0 0 80 48z">
+                    </path>
+                </svg>
```
Comment (msdeshmukh009) [inline src/components/Cart-Item-card/CartitemCard.jsx:69]: Is it possible to extract these SVGs as separate components or something? so jsx will become clean. You can also use react-icons.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good to me.

**Review by percius47 — APPROVED** [review]
The main page hero image is not correctly showing up. the vertical card images could also be positioned better. Rest looks good to go.

**Pull request merged**